src/fileio/fileio_party.cpp

```diff
@@ -382,60 +382,62 @@ bool load_party_v2(fs::path file_to_load, cUniverse& real_univ, bool manual){
 			// Load scenario data
 			std::istream& fin = partyIn.getFile("save/scenario.txt");
 			if(!fin) {
-				if(manual) showError("Loading Blades of Exile save file failed.");
+				if(!preview) showError("Loading Blades of Exile save file failed.");
 				return false;
 			}
 			file.readFrom(fin);
 			univ.scenario.readFrom(file);
 		}
 		
-		{ // Then the "setup" array
-			std::istream& fin = partyIn.getFile("save/setup.dat");
-			if(!fin) {
-				if(manual) showError("Loading Blades of Exile save file failed.");
-				return false;
-			}
-			file.readFrom(fin);
-			auto& page = file[0];
-			for(size_t i = 0; i < univ.party.setup.size(); i++) {
-				page[std::to_string(i)].extract(univ.party.setup[i]);
+		if(!preview){
```

Comment:
I would like it if you avoided adding a level of indentation here. It does mean redundantly checking `if(preview)` a few times instead of just once, but the optimizer should take care of that.

Reply:
I can also get rid of the extra indentation by bringing the loading of exported graphics *up*, and then checking if(preview) for an early return.

This should be fine as long as there's not a dependency between loading exported graphics and the things that I'm skipping. Which is something I'm already assuming, by skipping those things for the preview load.